tests/garage/tf/core/test_networks.py

```diff
@@ -0,0 +1,75 @@
+import tensorflow as tf
+from garage.tf.core.networks import mlp
+import numpy as np
```

Comment:
Please group imports and reorder them. There should be error msgs in CI.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Please fix this.